virtcontainers/pkg/vcmock/pod.go

```diff
@@ -18,28 +18,28 @@ import (
 	vc "github.com/kata-containers/runtime/virtcontainers"
 )
 
-// ID implements the VCPod function of the same name.
-func (p *Pod) ID() string {
+// ID implements the VCSandbox function of the same name.
```

Comment:
This file needs to be renamed to `sandbox.go`

Reply:
Done.

Reply:
thx!